tests/e2e/Login.test.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect, Page } from "@playwright/test";
+
+let page: Page;
+
+test.beforeEach(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto("/");
+
+  // logout first
+  const logout = await page.locator("a", { hasText: "Logout" });
+  await logout.click();
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe("playwright e2e test for login page", () => {
+  test("success login", async () => {
+    const email = await page.locator("#email");
+    const password = await page.locator("#password");
+    const submitButton = await page.locator("button[type=submit]");
+
+    await email.clear();
+    await email.type("eve.holt@reqres.in");
+    await password.clear();
+    await password.type("cityslicka");
+    await submitButton.click();
+
+    await expect(page).toHaveURL(/.*home/);
+  });
+
+  test("fail login", async () => {
+    const email = await page.locator("#email");
+    const submitButton = await page.locator("button[type=submit]");
+
+    await email.clear();
+    await email.type("test@123123abcabc");
+    await submitButton.click();
+    await page.waitForTimeout(5000);
```

Comment:
```suggestion
    await page.waitForTimeout(3000);
```
@G-N555 Looks like it could be a little shorter, what do you think?

https://github.com/monstar-lab-oss/reactjs-boilerplate/blob/854c962d40a1a58e1fb829b33d98292984db89d5/__mocks__/api/login.ts#L9

Reply:
@jinmayamashita
Yeah, I think should be ok. 
As you know, the biggest concern is that next action trigger before finish line40, so would be ok as long as `page.waitForTimeout` gives enough time for avoiding that.